### DISCUSSION TITLE
curious newbie question, no trolling intended

### DISCUSSION THREAD

**Discussion by sprappcom** [dop251/goja#581] [Q&A]
i saw this benchmark
```
The execution times roughly were:
otto: 33.195s
goja: 3.937s
duktape: 1.545s
v8 (go binding): 0.309s
v8 native (d8): 0.187s
```

ok so v8 go binding is faster by 10x. now the question is, what's the advantage of goja then? i also understand most golang v8 bindings are not frequently maintained but possible to provide some insights why people will still go with goja instead of binding?

**Comment by neimanpinchas** [comment]
As mentioned in the docs, goja is better in gases where a lot of interop is needed between the golang and javascript worlds, these things have a greater overhead in v8.
Goja will opn many times operate on the go object directly, not clone it.
